### PR TITLE
Upgrade lts version to 11.10.8

### PR DESCRIPTION
**Pull request by tmortagne**

### CODE REVIEW

File: library/xwiki [hunk 1]
```diff
@@ -4,14 +4,14 @@
 Maintainers: XWiki Core Dev Team <committers@xwiki.org> (@xwiki)
 GitRepo: https://github.com/xwiki-contrib/docker-xwiki.git
 
-Tags: 11, 11.10, 11.10.5, 11-mysql-tomcat, 11.10-mysql-tomcat, 11.10.5-mysql-tomcat, lts-mysql-tomcat, lts-mysql, lts
+Tags: 11, 11.10, 11.10.5, 11-mysql-tomcat, 11.10-mysql-tomcat, 11.10.7-mysql-tomcat, lts-mysql-tomcat, lts-mysql, lts
```
Comment (yosifkit) [inline library/xwiki:7]: MIssed a tag change: `11.10.5`

Comment (tmortagne) [reply library/xwiki:7]: Indeed, fixing that thanks.

**Comment by yosifkit** [comment]
Does maven not publish some way to verify downloads like a sha256sum or detached signature? (https://github.com/docker-library/official-images#security)

Alternatively, MySQL themselves publish a deb and signature (https://dev.mysql.com/downloads/connector/j/).

```console
$ sqlFile='mysql-connector-java_8.0.21-1debian10_all.deb'
$ wget "https://dev.mysql.com/get/Downloads/Connector-J/$sqlFile"
$ wget -O "$sqlFile.asc" "https://dev.mysql.com/downloads/gpg/?file=$sqlFile&p=3"
$ export GNUPGHOME="$(mktemp -d)"
$ gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys 'A4A9 4068 76FC BD3C 4567  70C8 8C71 8D3B 5072 E1F5'
$ gpg --batch --verify "$sqlFile"{.asc,}
```

**Comment by tianon** [comment]
> Does maven not publish some way to verify downloads like a sha256sum or detached signature? (https://github.com/docker-library/official-images#security)
> 
> Alternatively, MySQL themselves publish a deb and signature (https://dev.mysql.com/downloads/connector/j/).
> 
> ```
> $ sqlFile='mysql-connector-java_8.0.21-1debian10_all.deb'
> $ wget "https://dev.mysql.com/get/Downloads/Connector-J/$sqlFile"
> $ wget -O "$sqlFile.asc" "https://dev.mysql.com/downloads/gpg/?file=$sqlFile&p=3"
> $ export GNUPGHOME="$(mktemp -d)"
> $ gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys 'A4A9 4068 76FC BD3C 4567  70C8 8C71 8D3B 5072 E1F5'
> $ gpg --batch --verify "$sqlFile"{.asc,}
> ```

**Comment by vmassol** [comment]
@yosifkit hi. Hope you're doing well. Do we need to do something to have XWiki 11.10.8 docker build published? Thanks.

**Comment by tianon** [comment]
Yeah, we need to resolve https://github.com/docker-library/official-images/pull/8360#issuecomment-659000329 -- that MySQL connector download needs some sort of verification (GPG, checksum, or both).

**Comment by vmassol** [comment]
@tianon thanks I understand now (I had forgotten that I added a `curl` call to download binaries). I've now added a sha256 verification (for now).

**Comment by yosifkit** [comment]
Looks reasonable to me. :+1:

**Pull request merged**